Mage/src/main/java/mage/game/permanent/token/TokenImpl.java

```diff
@@ -218,8 +218,12 @@ public boolean putOntoBattlefield(int amount, Game game, Ability source, UUID co
         return putOntoBattlefield(amount, game, source, controllerId, tapped, attacking, attackedPlayer, attachedTo, true);
     }
 
-    @Override
     public boolean putOntoBattlefield(int amount, Game game, Ability source, UUID controllerId, boolean tapped, boolean attacking, UUID attackedPlayer, UUID attachedTo, boolean created) {
+        return putOntoBattlefield(amount, game, source, controllerId, tapped, attacking, attackedPlayer, attachedTo, created, Arrays.asList(this));
```

Comment:
is the list ever going to be modified? if not, cleaner to use Collections.singletonList()